docs/DeveloperGuide.adoc

```diff
@@ -312,58 +341,173 @@ Commands that are not undoable are not added into the `undoStack`. For example,
+
+
+image::CreatePersonListPanelSequenceDiagram.png[width="650"]
+
+
+[NOTE]
+If no  `EpicEvent` has been selected, `AttendeeList` will be empty
+
+==== Design Considerations
+
+When deciding on the UI, the following aspects of user experience were considered.
+
+===== Aspect: Overall UI Design
+
+- *Alternative 1 (current choice)*: A 3 pane UI consisting of list of persons, events, and attendees for selected event
+    * Pros: The user can view all 3 lists at the same time. He would know what arguments to supply when typing commands as he can read them off the list.
+    * Cons: The UI might become too cluttered as there are too many UI elements. However, given that EPIC is meant for modern computers with large displays, this should not be an issue.
+- *Alternative 2 (previous choice)*: 2 pane UI where the left pane is a 2 tab pane consistings of list of persons and events, and the right pane is the list of attendees.
+    * Pros: Merging the horizontal space for list of persons and events will create more space for list of attendees. The user is likely to be more interested in the attendees details.
+    * Cons: If the user needs to access data for some tabbed pane that is not in focus to fill out a command, this would break his workflow. The user will have to delete his current command, execute a command to set focus to the desired tab, memorize the required details and reenter his previous command.
+- *Alternative 3*: A common list that can display either list of persons, events or attendees for selected event
+    * Pros: We only have to make minimal changes to the UI layout.
+    * Cons: Events, persons, and attendees must be displayed using the same `Card` class. This would result in tight coupling of the display graphics logic  for the three lists.
 
-_{Explain here how the data encryption feature will be implemented}_
+=== [proposed] Export an Event
+
+The proposed export command exports the name of the attendees, their phone number, their email address, their home address and their attendance information of a particular event to a csv file into a file path specified by the user. Having the attendance of a particular event in a csv file would enable the users to analyse the data with other platforms.
+
+==== [proposed] Implementation
+
+The `export-event` command input is first parsed in `EventPlannerParser` to create an `ExportEventCommand`. The execution of it then calls the respective storage components. The subsequent export mechanism is handled by a `CsvEventPlannerStorage` which is a kind of `EventPlannerStorage`. The event data is parsed by `CsvUtil` and then exported as file through `CsvFileStorage` using the file path specified by the user.
+
+The sequence diagram below shows how the csv event export is processed in the storage component.
+
+image::ExportEventSequenceDiagram.png[width="650"]
```

Comment:
In this diagram, you need to mention what method is being called in the storage manager class